x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -90,11 +90,11 @@ export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData
       ),
       sortable: true,
       render: (score: Group['max_anomaly_score']) => {
-        if (score === null) {
+        if (score === undefined) {
```

Comment:
I think it is worth adding an extra check in here for `bucket score === 0`, and if so, displaying `0` rather than the current `< 1`. So if there are no anomalous buckets in the last 24 hours, it uses a different display to the case where there is a bucket score `0 < anomaly score < 1`

![image](https://user-images.githubusercontent.com/7405507/67082261-b2459500-f190-11e9-8445-b455948ce192.png)

Reply:
Updated here: 6da387db2ddce06eb80bca19b99432e8c3c63c2c